### PR TITLE
Clean up from wasm and common packages

### PR DESCRIPTION
**Pull request by vishniakov-nikolai**
### Details:
 - *item1*
 - *...*

### Tickets:
 - *ticket-id*

### CODE REVIEW

File: samples/js/node/README.md
Comment (almilosz) [inline samples/js/node/README.md:1]: In samples/js/node/.eslintrc.js there is a path to a `.eslintrc-common.js`. Common part is removed so I think this file can be renamed to `.eslintrc.js`

File: src/bindings/js/docs/README.md
Comment (almilosz) [inline src/bindings/js/docs/README.md:1]: The name of the module in tsconfig.json is "CommonJS"

File: src/bindings/js/CMakeLists.txt [hunk 1]
```diff
@@ -1,10 +0,0 @@
-# Copyright (C) 2018-2023 Intel Corporation
-# SPDX-License-Identifier: Apache-2.0
-#
-
-project(OpenVINO_JS_API)
-
-# if(EMSCRIPTEN)
-    add_subdirectory(wasm)
```
Comment (almilosz) [inline src/bindings/js/CMakeLists.txt:8]: Shouldn't we remove only the line?

**Review by almilosz — APPROVED** [review]
Looks good

File: samples/js/node/README.md
Comment (vishniakov-nikolai) [reply samples/js/node/README.md:1]: I renamed eslint-common to eslint-global to do not mix the name with package

File: src/bindings/js/docs/README.md
Comment (vishniakov-nikolai) [reply src/bindings/js/docs/README.md:1]: Thx, removed

File: samples/js/node/package.json
[the remark this answers is not in the document]
Comment (vishniakov-nikolai) [reply samples/js/node/package.json:1]: Updated)

**Pull request merged**